src/DbUtils.php

```diff
@@ -1820,7 +1820,7 @@ public function getUserName($ID, $link = 0, $disable_anon = false)
 
                     $user_params = [
                         'id'                 => $ID,
-                        'user_name'          => $username,
+                        'user_name'          => $username
```

Comment:
When your PR is finished, please, read the diff and remove changes that are useless. This kind of changes will pollute the "blame" view and the history of the file.

```suggestion
                        'user_name'          => $username,
```